x-men/frontend/public/template/menu.html

```diff
@@ -7,17 +7,34 @@
     <script src="{{scriptUri}}" defer></script>
 </head>
 <body>
-    <h1>SARA</h1>
-    <div class="options-wrapper">
-        <div>
-            <label for="location">Location:</label>
-            <select name="location" id="location">
-                <option value="local" selected="selected">Local</option>
-                <option value="remote">Remote</option>
-            </select>
-        </div>
+    <img src="{{imageUri}}sara.png" alt="SARA logo">
+    <div class="wrapper">
+        <div class="options-wrapper">
+            <div class="option-wrapper">
+                <div class="info-label-wrapper">
+                    <div class="tooltip"> &#9432;
+                        <span class="tooltiptext">
+                            <p>Selects which repository will be analyzed.</p>
+                            <ul>
```

Comment:
<p> element is not permitted as content under <span>html-validate(element-permitted-content)
<ul> element is not permitted as content under <span>html-validate(element-permitted-content)

Maybe change span to div?